app/views/supervisor_mailer/weekly_digest.html.erb

```diff
@@ -7,7 +7,11 @@
   </tr>
   <tr style="font-family: Helvetica,Arial,sans-serif; box-sizing: border-box; margin: 0;">
     <td class="content-block" style="font-family: Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 16px; vertical-align: top; margin: 0; padding: 0 0 20px;" valign="top">
+    <% if @supervisor.volunteers.length == 0 %>
+      You have no volunteers with assigned cases at the moment. When you do, you will see their status here.
```

Comment:
@compwron Let me know if this text should be updated.

Reply:
seems good!